design/20220118.certificate-issuance-exponential-backoff.md

```diff
@@ -73,14 +73,14 @@ Currently failed issuances are retried once an hour without a backoff or time li
 
 ## Proposal
 
-Exponential backoff will be implemented by exponentially increasing the delays between a failed issuance ([`Issuing` condition set to false in `certificates-issuing` controller](https://github.com/jetstack/cert-manager/blob/196d0011ca46037186a826365bcd6316d9b9462a/pkg/controller/certificates/issuing/issuing_controller.go#L341)) and a new issuance ([`Issuing` condition set to true in `certificates-trigger` controller](https://github.com/jetstack/cert-manager/blob/d5503c2ed2df272ec1bd94ebd223408fad29df1f/pkg/controller/certificates/trigger/trigger_controller.go#L184)). From a user perspective, this will correspond to the delay between a `CertificateRequest` having failed and new `CertificateRequest`s being created.
```

Comment:
Eventually jetstack could remove their fork, and there's no particular reason to keep the reference as the code is present here...

Reply:
I agree with changing the urls. To clarify: github.com/jetstack/cert-manager is not a fork, it was the original location of the cert-manager repo, it was moved after being donated to the CNCF.